Firestore/Example/Firestore.xcodeproj/xcshareddata/xcschemes/Firestore_Tests.xcscheme

```diff
@@ -20,27 +20,12 @@
                ReferencedContainer = "container:Firestore.xcodeproj">
             </BuildableReference>
          </BuildActionEntry>
-         <BuildActionEntry
```

Comment:
Probably no reason for this file to have changed other than me opening it in Xcode 9.